docs/introduction/faq.md

```diff
@@ -0,0 +1,109 @@
+
+## Is the application https ready or does it need some extra setting?
+Shopys Framework is fully prepared for HTTPS.
+
+## What about translations and language constants export and import?
+To set up the user translations of labels and messages, use the files `messages.cs.po` and `validators.cs.po`, where `cs` represents the version of translations for a locale `cs`.
+These files are generated for each locale you use, and you can find them in a directory `ShopBundle/Resources/translations/`.
+Changes and language settings are described more in detail in the tutorial [How to Set Up Domains and Locales (Languages)](./how-to-set-up-domains-and-locales.md#3-locale-settings).
+
+## What about deploy and production server setting?
+We recommend installation using the Docker for production.
+See how to install Shopsys Framework in production and how to proceed when deploying in the tutorial [Installation Using Docker on Production Server](../installation/installation-using-docker-on-production-server.md).
+
+## How to set up the administration with a Czech locale?
+The administration is by default in `en` locale.
+If you want to switch it to the another locale, override the method `getAdminLocale()` of the class `Shopsys\FrameworkBundle\Model\Localization\Localization`.
+This scenario is described in more detail in the tutorial [How to Set Up Domains and Locales (Languages)](./how-to-set-up-domains-and-locales.md#36-locale-in-administration).
+
+## What are the differences between "listable", "sellable", "offered" and "visible" products?
+Products can be grouped into several groups according to their current status or according to what they are used for.
+
+**Visible** - products that appear in the database as visible. 
+The conditions which the product must satisfy to appear as visible:
+- the product must not be set as hidden
+- if the attribute "selling start date" is filled in, the value of this attribute must be set to the date in the past
+- if the attribute "selling end date" is filled in, the value of this attribute must be set to the date in the future
+- the product must have a name for the specific locale
+- if the product is a variant, there must exist calculated price for this variant for the specific pricing group
+- if the product is a variant, her main variant must not be set as hidden
+- if the product is the main variant, at least one of her variants must be visible
+
+**Offered** - products that satisfied the conditions for **visible** and at the same time they appear in the database with the attribute `calculatedSellingDenied` = `FALSE`.
+The `calculatedSellingDenied` attribute shows whether the product is already sold out or if the product is a variant with the main variant that is not set up with selling denied.
+
+**Listable** - products that satisfied the conditions for **offered** and at the same time, these products are not the variants.
+By default, the specific variants are not included in the product lists.
```

Comment:
What are you trying to say here?

Reply:
I think you have said it in the previous sentence did you not?

Reply:
Done